### PR TITLE
ENH: Enable pre-commit CI autofixes

### PR DESCRIPTION
**Pull request by fepegar**
I'm going to introduce a formatting issue next, to make sure the bot fixes it.

### CODE REVIEW

**Comment by fepegar** [comment]
Looks like it's working.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -3,7 +3,6 @@
 # See https://github.com/pre-commit/pre-commit-hooks/blob/master/.pre-commit-config.yaml for an example with more hooks
 
 ci:
-  autofix_prs: false
   autoupdate_commit_msg: "ENH: Autoupdate hooks [pre-commit.ci]"
```
Comment (vale-salvatelli) [inline .pre-commit-config.yaml:6]: if this is the commit that happens in the PR I don't think it should have ENH in front - maybe better something like '[pre-commit.ci] pre-commit fixes'?

Comment (fepegar) [reply .pre-commit-config.yaml:6]: It's actually the default PR title for the autoupdates, not for commits. We can change the PR prefix to `MNT` after #224.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -3,7 +3,6 @@
 # See https://github.com/pre-commit/pre-commit-hooks/blob/master/.pre-commit-config.yaml for an example with more hooks
 
 ci:
-  autofix_prs: false
```
[the remark this answers is not in the document]
Comment (fepegar) [reply .pre-commit-config.yaml:6]: Resolved in a08172d.

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@
...
+  autofix_prs: true
   autoupdate_commit_msg: "ENH: Autoupdate hooks [pre-commit.ci]"
   autoupdate_schedule: monthly
```
Comment (ant0nsc) [inline .pre-commit-config.yaml:8]: do we still need the monthly update?

**Review by mebristo — APPROVED** [review]

File: .pre-commit-config.yaml (hunk 3 above)
Comment (fepegar) [reply .pre-commit-config.yaml:8]: It's like Dependabot: it looks for newer versions of the hooks monthly (it doesn't run `pre-commit` monthly).

Comment (fepegar) [reply .pre-commit-config.yaml:8]: So, we don't _need_ it, but I think it's a good feature to enable.

**Pull request merged**